### PR TITLE
Update/masterportalapi

### PR DESCRIPTION
**Pull request by dopenguin**
## Summary

Updated `@masterportal/masterportalApi` and subsequently `ol` to the latest version; `ol` has released a new major while implementing this version upgrade and another update should be done once `@masterportal/masterportalApi` upgrades, as the update doesn't seem to be too big.

- Adjusted `jest` setup to fix new transform-issues
- Upgraded node-version to current LTS
- Removed now redundant postinstall script overriding the `@masterportal/masterportalApi`; a relevant change has been added to `@polar/plugin-address-search`
- Updated `cesium` dependency to `@cesium/enginge` inline with `@masterportal/masterportalApi`
- Changed some `@masterportal/masterportalApi` imports as `rawLayerList` has been changed to use a default export instead of a named export
- Shortened some imports `@masterportal/masterportalApi` by removing the `src` part as they are exposed directly now
- Fixed types related to `VectorLayer` as they now need the argument `Feature` instead of `VectorSource`
- Adjusted `vite` configuration to fix issue with `olcs` exports

## TODOs

- [x] Add correct changelog entries (some are breaking, some not)
- [x] Add / move `ol` dependencies to / in some packages

## Instructions for local reproduction and review

All clients need to be tested whether they still behave in the correct manner.

### CODE REVIEW

File: jest.setup.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import 'regenerator-runtime/runtime'
```
[the remark this answers is not in the document]
Comment (dopenguin) [reply jest.setup.js:1]: Sounds good, will test it later!

[the remark this answers is not in the document]
Comment (dopenguin) [reply jest.setup.js]: Done did it ad1bcbf8d15c58253b647b6e3c6c84f56db39f68

**Comment by dopenguin** [comment]
I've also adjusted the github actions 1ca9f8c01991eab0fde79ec1bcceea254704c21a

**Comment by dopenguin** [comment]
🏓 @warm-coolguy 

I've added all the changelog entries. Do you agree that all plugins should get a major update as the peerDependency is changing with a major or do you think this can be neglected as the api of the plugin itself hasn't changed?

**Review by warm-coolguy — APPROVED** [review]
>🏓 @warm-coolguy 
>
>I've added all the changelog entries. Do you agree that all plugins should get a major update as the peerDependency is changing with a major or do you think this can be neglected as the api of the plugin itself hasn't changed?

I deem it appropriate to model this as a breaking change as consumers may have implemented things on top of our `ol` version. E.g. I know of a client that does further work on geometries, which may use the bundled `ol` somehow.

🚀 @dopenguin

**Pull request merged**